src/pages/payment-methods.jsx

```diff
@@ -0,0 +1,152 @@
+import Image from "next/image";
+import Link from "next/link";
+import { withTranslation } from "next-i18next";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import Carousel from "react-multi-carousel";
+
+import "react-multi-carousel/lib/styles.css";
+
+import PageIntro from "@/components/PageIntro";
+import Button from "@/components/ui/Button";
+
+import MasterCardSVG from "/public/images/master-card.svg";
+import VisaSVG from "/public/images/visa.svg";
+function PaymentMethod({ t }) {
+    const responsive = {
+        desktop: {
+            breakpoint: { max: 3000, min: 1024 },
+            items: 3,
+        },
+        tablet: {
+            breakpoint: { max: 1024, min: 464 },
+            items: 2,
+        },
+        mobile: {
+            breakpoint: { max: 464, min: 0 },
+            items: 1,
+        },
+    };
+    const colors = [
+        { color: "#e62151", bg: "pink" },
+        { color: "#1d40c0", bg: "blue" },
+        { color: "#daa00a", bg: "yellow" },
+    ];
+    const cards = [
+        {
+            type: "mastercard",
+            icon: MasterCardSVG,
+            cardNumber: "xxxx xxxx xxxx 2000",
+            cvv: "123",
+            expireDate: "02/2020",
+            holder: "Abrar",
+        },
+        {
+            type: "visa",
+            icon: VisaSVG,
+            cardNumber: "xxxx xxxx xxxx 1990",
+            cvv: "113",
+            expireDate: "02/2030",
+            holder: "Maha",
+        },
+        {
+            type: "mastercard",
+            icon: MasterCardSVG,
+            cardNumber: "xxxx xxxx xxxx 1980",
+            cvv: "133",
+            expireDate: "02/2040",
+            holder: "Mawaheb",
+        },
+    ];
+    return (
+        <>
+            <div className='container mt-12'>
+                <PageIntro
+                    title={t("selectCard")}
+                    subtitle={t("selectCardDesc")}
+                />
+                <Carousel
+                    infinite='true'
+                    className='mt-28'
+                    responsive={responsive}
+                >
+                    {cards.map((item, index) => {
+                        return (
+                            <>
```

Comment:
- There is no need for this `tag` because it causes `st should have a unique "key" prop.
`

Reply:
@dhiashalabi this PR is draft and i'm still waiting for your customizing radiogroup to put everything well